src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -405,20 +405,20 @@ static ucs_status_t uct_rc_mlx5_iface_preinit(uct_rc_mlx5_iface_common_t *iface,
     /* Multi-Packet XRQ initialization */
     if (!ucs_test_all_flags(md->flags, UCT_IB_MLX5_MD_FLAG_MP_RQ       |
                                        UCT_IB_MLX5_MD_FLAG_DEVX_RC_SRQ |
-                                       UCT_IB_MLX5_MD_FLAG_DEVX_RC_QP)) {
+                                       UCT_IB_MLX5_MD_FLAG_DEVX_RC_QP) ||
+        (mlx5_config->tm.mp_num_strides == 1)) {
         return UCS_OK;
     }
 
-    if ((mlx5_config->tm.mp_num_strides == UCS_ULUNITS_AUTO) ||
-        (mlx5_config->tm.mp_num_strides == 1)) {
-        return UCS_OK;
-        /* TODO: make the following to be default when MP support is added to UCP
-        iface->tm.mp.num_strides = UCS_BIT(IBV_DEVICE_MP_MIN_LOG_NUM_STRIDES); */
+    if (mlx5_config->tm.mp_num_strides == UCS_ULUNITS_AUTO) {
+        iface->tm.mp.num_strides = 8;
```

Comment:
would it be better to define a constant for this and show the default to user?
```
#define UCT_RC_MLX5_TM_MP_NUM_STRIDES 8
```
```
{"TM_NUM_STRIDES", UCS_PP_MAKE_STRING(UCT_RC_MLX5_TM_MP_NUM_STRIDES),
...
```
for me, `auto` means that it can be defined in runtime only - some device characteristics or etc.

Reply:
we support 16 as well
originally I had a set of constants, but there was a comment to remove them

Reply:
ah, ok

Reply:
agree we should have 8 there
if need another var to enable/disable striding RQ altogether it can be yes/no/try

Reply:
Using 8 instead of auto as default value is confusing.
MP XRQ is not always supported, so it may be 1 in some cases

Reply:
i think we should separate:
MP_SRQ_ENABLE=yes/no/try (default: try)
MP_SRQ_NUM_STRIDES=<number> (default: 8)

i think having them bot in same var, like today, is more confusing

Reply:
should I go this way?
imo, having one var is better

Reply:
yes, from user perspective it's simpler to understand IMHO